### DISCUSSION TITLE
will mojo be able to build a processing farm across machines on a network?

### DISCUSSION THREAD

**Discussion by j215chic** [modular/modular#906] [General]
So I have ( and want to use for this test) the following environment. 

running linux mint on my trusty HP Z840 with an old ( YES I KNOW)  Quadro k5000 (1536 cores) The z840 has dual E5-2699 = 35 cores and a jetson nano ( original). I am running a cat 6 network.  The z840 has 2 lan ports : Intel Ethernet I218-LM and a  Intel I210 Gigabit.

Next month I am starting a new project and want to use mojo and python. I'll be using pandas 2.0 and polars.  

**So my question is 

"if I hook up my nano to the back of the z840 using a cat 6 cable can mojo be configured to 'scan' my z840 AND the nano and arrive at a "core farm" that I can use abstractly in my coding?"**  

I just want to concentrate on the coding and have the compiler do the heavy lifting as to which device does what.  Thanks for any help, this is VERY exciting.

**Comment by lattner** [comment]
Mojo is a programming language, but you could build libraries on top of it to provide this functionality.

**Comment by j215chic** [comment]
my mistake I was addressing modular. Wrong forum sorry.
https://youtu.be/pdJQ8iVTwj8?t=3613
